cni/pkg/ambient/net.go

```diff
@@ -220,7 +238,7 @@ func GetHostIP(kubeClient kubernetes.Interface) (string, error) {
 func (s *Server) AddPodToMesh(pod *corev1.Pod) {
 	switch s.redirectMode {
 	case IptablesMode:
-		AddPodToMesh(s.kubeClient.Kube(), pod, "")
+		_ = AddPodToMesh(s.kubeClient.Kube(), pod, "")
```

Comment:
Would be good to import your PR comment about why the error is ignored to this spot for future spelunkers.

Reply:
Sure. 

Maybe in next meeting we should discuss the status and plans for the "post-start injection" - I have some concerns and it may be better to skip it from the initial beta or release.  It can be useful for migrations ( or not - it does
break existing connections and is hard to operate ) - but it adds to the test and support burdens, and is not strictly required. For example ignoring the other pod IPs is easy to fix - but still work and testing, and who knows what other gaps exist. We can bring it back later.